lib/config-generator.js

```diff
@@ -558,6 +585,10 @@ class ConfigGenerator {
             splitChunks
         );
 
+        if (this.webpackConfig.useIntegrity > 0) {
```

Comment:
why `> 0` while it is a boolean ?

Reply:
Bad search/replace, before introducing `webpackConfig.useIntegrity`, I've used `webpackConfig.integrityAlgorithms.length > 0` to check if integrity was enabled.